CODE_GENERATION.md

```diff
@@ -1,16 +0,0 @@
-# Code Generation
```

Comment:
I assume this got deleted b/c it's somewhere else now?

Reply:
Yes. The styra go client has been moved into it's own library: https://github.com/MisterMX/styra-go-client